CanvasCore/CanvasCore/Modules/ModuleItem/ModuleItem+Network.swift

```diff
@@ -54,7 +54,7 @@ extension ModuleItem {
     public static func moduleItemSequence(_ session: Session, courseID: String, moduleItemID: String) throws -> SignalProducer<JSONObject, NSError> {
         let path = "api/v1/courses/\(courseID)/module_item_sequence"
         let parameters = ["asset_id": moduleItemID, "asset_type": "ModuleItem"]
-        let request = try session.GET(path, parameters: parameters)
+        let request = try session.GET(path, parameters: parameters, paginated: false)
```

Comment:
Could this cause any performance issues for large courses?

Reply:
This particular api call isn't a paginated list at all. `session.GET` is dumb and will always append `per_page=99` unless we tell it not to.